src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt

```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\Users\mamirzam\CV-API\azure-sdk-for-net\src\SDKs
-2018-12-11 18:58:56 UTC
+cmd.exe /c autorest.cmd https://github.com/toothache/azure-rest-api-specs/blob/ocr_read/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\Source\cognition\azure-sdk-for-net\src\SDKs
```

Comment:
Will generate SDK using code from master branch of azure-rest-api-specs once the [PR](https://github.com/Azure/azure-rest-api-specs/pull/5314) is approved and gets merged.

[Computer Vision] Add read to CV OCR API. #5314